packages/react-devtools-shared/src/backend/index.js

```diff
@@ -56,6 +64,14 @@ export function initBackend(
   ];
 
   const attachRenderer = (id: number, renderer: ReactRenderer) => {
+    // skip if already attached
+    if (renderer.attached) {
+      return;
+    }
+    // only attach if the renderer is compatible with the current version of the backend
+    if (!isMatchingRender(renderer.reconcilerVersion || renderer.version)) {
+      return;
```

Comment:
Should this emit the `unsupported-renderer-version` on line 103?

Reply:
No, `unsupported-renderer-version` is meant to detect React older than v15. See UnsupportedVersionDialog.js
This new backend model will only match one version (with exception of react-devtools-backend-compact that supports v15~v18). 
This is a good point I should update some event names / variables / comments to increase code readability